fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/service/AccountDetailsReadPlatformServiceJpaRepositoryImpl.java

```diff
@@ -131,6 +162,16 @@ public AccountSummaryCollectionData retrieveGroupAccountDetails(final Long group
         return retrieveLoanAccountDetails(loanWhereClause, new Object[] { clientId });
     }
 
+    @Override
+        public List<LoanAccountSummaryData> retrieveLoanAccountDetailsByGroupIdAndGlimAccountNumber(final Long groupId,final String glimAccount) {
+        final LoanAccountSummaryDataMapper rm = new LoanAccountSummaryDataMapper();
+        final String loanWhereClauseForGroupAndLoanType = " where l.group_id =? and glim.account_number="+glimAccount+" and l.loan_type_enum=4";
```

Comment:
can we pass GLIM account number as we are passing group id, appending it in string doesn't look good.